dashboard/db/migrate/20160213001925_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+  end
```

Comment:
Are you migrating the existing submitted levels? I think that should be done here.

To get an estimate of how long the migration will take, you can run this on the existing reporting read replica (and then drop the column so that it doesn't blow up when the real db is migrated for real). Can you coordinate with @ashercodeorg to find a time when the read replica won't be used?